.github/workflows/ci.yml

```diff
@@ -27,11 +27,11 @@
 # - You can investigate various information at the right pane of GitHub
 #   Actions report page.
 #
-#   | Item                      | Section in the right pane             |
-#   | ------------------------- | ------------------------------------- |
-#   | OS, VM                    | Set up job                            |
-#   | git repo, commit hash     | Run actions/checkout@v3               |
-#   | gcc, tools                | Environment info                      |
+#   | Item                      | Section in the right pane                                              |
+#   | ------------------------- | ---------------------------------------------------------------------- |
+#   | OS, VM                    | Set up job                                                             |
+#   | git repo, commit hash     | Run actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab # v3.5.2 |
```

Comment:
Is it useful to also update the comment ?

Reply:
Up to you! We can revert to `Run actions/checkout@v3` or change to `Run actions/checkout@v3.5.2`, whatever feels easier to understand here.

Reply:
Indeed, it's just an overview,
I would assume `Run actions/checkout@v3` to be significant enough here.

I'm just concerned about the future, aka triggering automated code changes, like dependabot, that could miss the comment nature of this line.

Reply:
I have tested renovatebot on this case. The bot updates only the real action line and not the table comment line. So, the table comment and the action may get unsynced over time.

Reply:
Complementing, I have tested comment lines such as:
- # | Run actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab # v3.5.2 |
- # actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab
- # actions/checkout@v3
and none get updated by renovatebot.

Reply:
Yes, that's why I was more in favor of keeping it simple and generic, since it won't get updated with the rest of the code.
But don't worry, I'll do it directly.